### PR TITLE
LPS-151747 Create an AssetVocabulary item selector

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-151747

### CODE REVIEW

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:reevaluate:1377401_1801

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:relevant

File: modules/apps/asset/asset-vocabulary-item-selector-web/src/main/resources/META-INF/resources/select_asset_vocabulary.jsp [hunk 1]
```diff
@@ -84,4 +93,47 @@ SelectAssetVocabularyItemSelectorDisplayContext selectAssetVocabularyItemSelecto
 			markupView="lexicon"
 		/>
 	</liferay-ui:search-container>
-</aui:form>
\ No newline at end of file
+</aui:form>
+
+<c:if test="<%= !selectAssetVocabularyItemSelectorDisplayContext.isMultiSelection() %>">
```
Comment (victorg1991) [inline modules/apps/asset/asset-vocabulary-item-selector-web/src/main/resources/META-INF/resources/select_asset_vocabulary.jsp:98]: Can we move this to a js file? Like in other places: https://github.com/liferay/liferay-portal/blob/4ba6dd799775995c08e00beab148fafc01be2828/modules/apps/asset/asset-categories-item-selector-web/src/main/resources/META-INF/resources/js/SelectEntityHandler.js

Also, I'd change all the `var` to `const` 😇

**Comment by victorg1991** [comment]
@lfbesada Nice 👏  I managed to try it and it is working fine. Just a minor comment to see if we can improve a bit the code and make it easier to debug in the future 😇

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ealonso** [comment]
see: https://github.com/liferay-echo/liferay-portal/pull/7890

**Pull request closed without merging**